package.json

```diff
@@ -2,64 +2,80 @@
   "name": "marksy",
   "version": "6.0.2",
   "description": "Convert markdown into react components",
-  "main": "lib/index.js",
-  "scripts": {
-    "start": "webpack-dev-server --devtool eval-source-map --progress --colors --content-base build",
-    "prepare": "npm run build",
-    "build": "BABEL_ENV=production babel src/ --out-dir=lib/ -s",
-    "test": "jest && size-limit",
-    "size": "size-limit"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/cerebral/marksy"
-  },
   "keywords": [
-    "react",
+    "convert",
     "markdown",
-    "convert"
+    "react"
   ],
-  "author": "Christian Alfoni",
-  "license": "MIT",
+  "homepage": "https://github.com/storybooks/marksy",
   "bugs": {
-    "url": "https://github.com/cerebral/marksy/issues"
+    "url": "https://github.com/storybooks/marksy/issues"
   },
-  "homepage": "https://github.com/cerebral/marksy",
-  "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-core": "^6.24.1",
-    "babel-jest": "^19.0.0",
-    "babel-loader": "^7.0.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "css-loader": "^0.15.2",
-    "highlight.js": "^9.12.0",
-    "inferno-create-element": "^3.3.1",
-    "inferno-server": "^3.3.1",
-    "jest": "^20.0.1",
-    "json-loader": "^0.5.2",
-    "node-libs-browser": "^0.5.2",
-    "preact": "^8.1.0",
-    "preact-render-to-string": "^3.6.3",
-    "prismjs": "^1.6.0",
-    "react": "^15.5.4",
-    "react-dom": "^15.5.4",
-    "react-test-renderer": "^15.5.4",
-    "regenerator-runtime": "^0.10.3",
-    "size-limit": "^0.13.1",
-    "style-loader": "^0.12.3",
-    "webpack": "^2.4.1",
-    "webpack-dev-server": "^2.4.2"
+  "license": "MIT",
+  "author": "Christian Alfoni",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/storybooks/marksy"
+  },
+  "scripts": {
+    "build": "BABEL_ENV=production babel src/ --out-dir=lib/ -s",
+    "lint": "cross-env NODE_ENV=production eslint --cache --cache-location=.cache/eslint --ext .js,.jsx,.json ./src ./app",
```

Comment:
Let's add this as a circleci build step and a precommit hook

Reply:
YES!

Reply:
done

Reply:
The CI part, you mean?

Reply:
ye, I'll add lint-staged later as well